lib/Froxlor/Dns/Dns.php

```diff
@@ -144,7 +144,7 @@ public static function createDomainZone($domain_id, $froxlorhostname = false, $i
 			}
 			if (Settings::Get('dkim.use_dkim') == '1') {
 				// check for DKIM content later
-				self::addRequiredEntry('dkim_' . $domain['dkim_id'] . '._domainkey', 'TXT', $required_entries);
+				self::addRequiredEntry('main._domainkey', 'TXT', $required_entries);
```

Comment:
why hardcode the selector if froxlor already provides one which is unique for the key?

Reply:
I do not see the need for an unique selector...

Reply:
unique per key - a new key gets generated, you get a new selector. somewhat security related :) we should keep it

Reply:
That's an argument. I will revert.